exercises/tests/tests3.rs

```diff
@@ -1,5 +1,5 @@
 // tests3.rs
-// This test isn't testing our function -- make it do that in such a way that
+// This test isn't testing our function -- make it does that in such a way that
```

Comment:
"does" should be changed to "so"
"... make is so that in such a way ..."
or
"... make is so that the test passes. ..."

Reply:
I think "make it do so in such a way [...]" also works

Reply:
well, the way the desc is right now itself seems correct. 🤷

Reply:
Woops you're right I must've misread it! Never mind! :)